docs/tutorials/pre_executed/using_rubin_data.ipynb

```diff
@@ -0,0 +1,404 @@
+{
```

Comment:
<p>but even after you modify, the nDiaSources stays the same. It is not dynamically updated, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>